go.mod

```diff
@@ -411,6 +411,8 @@ require (
 
 replace (
 	bazil.org/fuse => bazil.org/fuse v0.0.0-20200117225306-7b5117fecadc // specific version for iOS building
+
+	berty.tech/weshnet => ../weshnet
```

Comment:
I am curious. Does this mean that berty will build with whatever version of weshnet that the developer has checked out locally in ../weshnet ?

Reply:
Yes, that allows us to test berty with the local version of weshnet and check that everything is good before pushing weshnet and berty.